Error compiling the current master branch under g++ 7.5

The following error occurs during `make -j4`. Note that an analogical error occurs when compiling the `cadical` dependency, too.

```/home/user/CVC4/src/./prop/cryptominisat.h:80:5: error: ‘TimerStat’ does not name a type; did you mean ‘SizeStat’?
     TimerStat d_solveTime;
     ^~~~~~~~~
     SizeStat
/home/user/CVC4/src/prop/cryptominisat.cpp: In member function ‘virtual CVC4::prop::SatValue CVC4::prop::CryptoMinisatSolver::solve()’:
/home/user/CVC4/src/prop/cryptominisat.cpp:169:3: error: ‘TimerStat’ has not been declared
   TimerStat::CodeTimer codeTimer(d_statistics.d_solveTime);
   ^~~~~~~~~

```

Using `./configure.sh production --abc --cadical --cryptominisat`

--- a/src/prop/cadical.h
+++ b/src/prop/cadical.h
@@ -22,6 +22,7 @@
 #ifdef CVC4_USE_CADICAL
 
 #include "prop/sat_solver.h"
+#include "util/stats_timer.h"
 
 #include <cadical.hpp>
 

--- a/src/prop/cryptominisat.h
+++ b/src/prop/cryptominisat.h
@@ -22,6 +22,7 @@
 #ifdef CVC4_USE_CRYPTOMINISAT
 
 #include "prop/sat_solver.h"
+#include "util/stats_timer.h"
 
 // Cryptominisat has name clashes with the other Minisat implementations since
 // the Minisat implementations export var_Undef, l_True, ... as macro whereas

--- a/src/prop/kissat.h
+++ b/src/prop/kissat.h
@@ -22,6 +22,7 @@
 #ifdef CVC4_USE_KISSAT
 
 #include "prop/sat_solver.h"
+#include "util/stats_timer.h"
 
 extern "C" {
 #include <kissat/kissat.h>

